.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,8 @@
 <!-- Put an x inside [ ] to check it, like so: [x] -->
 
 - [ ] If code changes were made then they have been tested.
-    - [ ] I have updated the documentation to reflect the changes.
+  - [ ] I have updated the documentation to reflect the changes.
+  - [ ] I have ran `task pyright` and fixed the relevant issues.
```

Comment:
This should be either "I ran" or "I have run". The past participle of "ran" is "run".

```suggestion
  - [ ] I have run `task pyright` and fixed the relevant issues.
```

Reply:
it was from https://github.com/nextcord/nextcord/pull/596#discussion_r848267473 sorry